jaeger-crossdock/src/test/java/com/uber/jaeger/crossdock/resources/behavior/http/TraceBehaviorResourceTest.java

```diff
@@ -82,12 +82,10 @@ public static void setUpLogger() {
 
   @Before
   public void setUp() throws Exception {
-    port = System.getenv("TRACE_TEST_BEHAVIOR_PORT");
```

Comment:
what was setting this env variable before & why do we not need it?  cc @black-adder